package.json

```diff
@@ -17,7 +17,9 @@
     "react-redux": "^6.0.1",
     "react-router": "^4.4.0",
     "react-router-dom": "^5.0.0",
+    "react-thunk": "^1.0.0",
```

Comment:
Why do you need it?

Reply:
that was added by mistake, i needed redux-thunk